projects/storefrontlib/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -35,13 +38,24 @@ import { map, startWith } from 'rxjs/operators';
   selector: 'cx-form-errors',
   templateUrl: './form-errors.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
```

Comment:
Do you think we could use `@HostBinding`s for these attributes like you did for `attr.role`? I think it could be easier to understand and modify.

Reply:
This is completely reasonable point. I was just confused seeing both. I think its better to be consistent with whichever was there first just in my opinion.